static/js/components/ExpandedPostDisplay_test.js

```diff
@@ -197,23 +197,6 @@ describe("ExpandedPostDisplay", () => {
     })
   })
 
-  //
-  ;[[true, "Unfollow"], [false, "Follow"]].forEach(
-    ([subscribed, buttonText]) => {
-      it(`should include a ${buttonText} button when subscribed === ${subscribed}`, () => {
-        post = makePost()
-        post.subscribed = subscribed
-        const wrapper = renderPostDisplay()
-        const button = wrapper.find(
-          subscribed ? ".subscribed" : ".unsubscribed"
-        )
-        assert.include(button.text(), buttonText)
-        button.simulate("click")
-        assert.ok(toggleFollowPostStub.called)
-      })
-    }
-  )
```

Comment:
might be good to add a test here just confirming that the right props are passed down to the `FollowButton` component

Reply:
✅